SSA/Core/Util.lean

```diff
@@ -109,3 +109,45 @@ instance [inst : Cli.ParseableType τ] {n : ℕ} : Cli.ParseableType (Vector τ
       return h ▸ Vector.ofArray arr
     else
       none
+
+
+def Std.BitVec.width : Std.BitVec w → Nat := fun _ => w
```

Comment:
```suggestion
@[simp] abbrev Std.BitVec.width : Std.BitVec w → Nat := fun _ => w
```
I expect we don't really care about having the function in goals, so having it automatically `simp` away is nice